prisma/sql/getProviderRetrievability.sql

```diff
@@ -0,0 +1,8 @@
+select
+    100*ceil(success_rate *20)/20 - 5 as "valueFromExclusive",
+    100*ceil(success_rate *20)/20     as "valueToInclusive",
+    count(*)::int                     as "count",
+    date_trunc('week', date)          as "week"
+from provider_retrievability_daily
```

Comment:
This is daily retrievability, we need to aggregate `success_rate` to week-windows first. Right now it will count the same provider 7 times (once for each day in a week).

Easiest solution: switch to `providers_weekly` table, which has `avg_retrievability_success_rate` field with correctly calculated retrievability score for given week & provider.